_config.yml

```diff
@@ -13,7 +13,7 @@
 # you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: Your awesome title
+title: Scout Airlines
```

Comment:
What is this config file for? the meta data?

Reply:
Yes! This config file in Jekyll acts as a meta data/ <title> of the page.

Reply:
okay awesome! thanks for the info 🙂